docs/go/workflows.md

```diff
@@ -42,4 +43,114 @@ A Workflow can have one or more such parameters.
+    Temporal provides support for both buffered and unbuffered channels.
+    - `workflow.Selector` This is a replacement for the `select` statement. Learn more on the [Go SDK Selectors](https://docs.temporal.io/docs/go/selectors) page
+3. Additionally, for executing very small pieces of nondeterministic logic within the Workflow, you can use the [`workflow.SideEffect` API](/docs/go/side-effect).
+
+Below is a sample Workflow that is treated as a cron job by the Temporal Server.
+It executes a single Activity and uses `workflow.Now()`.
+
+<!--SNIPSTART samples-go-cron-workflow-->
+<!--SNIPEND-->
+
+## How to start a Workflow
+
+With the Go SDK, there are two ways that you can start a Workflow:
+
+1. Use the Go SDK `client` to start a Workflow from a Go process, as described below.
+2. Start a Workflow from an already running Workflow, which is known as a [Child Workflow](/docs/go/child-workflows).
+
+:::note
+
+Starting a Workflow is not the same as executing a Workflow.
+Starting a Workflow means that you are telling the Server to begin tracking the state of the Workflow execution.
+In a Temporal application, you do not run Workflow code directly, instead Workflow code is hosted and executed by a [Worker](/docs/go/workers).
+
+:::
+
+To start a Workflow you need to create the Temporal Go SDK client, call `ExecuteWorkflow()`, and pass it the following:
+
+1. `context.Context`, which is a normal [Go `Context` type](https://golang.org/pkg/context/).
+2. `client.StartWorkflowOptions` Which can include [timeout settings and a `RetryPolicy`](/docs/go/retries).
+3. The name of the Workflow function.
+4. Variables that should be passed to the Workflow when it begins execution.
+
+```go
+package main
+
+import (
+    "context"
+
+    "go.temporal.io/sdk/client"
+)
+
+func main() {
+    // Create the client object just once per process
+    c, err := client.NewClient(client.Options{})
+    if err != nil {
+        // Handle failure
+    }
+    defer c.Close()
+
+    // TaskQueue is the only required option to start a Workflow
+    options := client.StartWorkflowOptions{
+        TaskQueue: "your_task_queue",
+    }
+
+    we, err := c.ExecuteWorkflow(context.Background(), options, YourWorkflow, workflowParm1, workflowParam2)
+    if err != nil {
+        // Handle failure
+    }
+
+    // Use the WorkflowExecution to get the result
+    // Get is blocking call and will wait for the Workflow to complete
+    var workflowResult string
+    err = we.Get(context.Background(), &workflowResult)
+    if err != nil {
+        // Handle failure
+    }
+
+    // Do something with the result
+    printResults(workflowResult, we.GetID(), we.GetRunID())
+}
+```
+
+In Go, the only difference between a synchronous start and an asynchronous start is whether you use the `WorkflowExecution` that is returned from `ExecuteWorkflow` to get the result of the Workflow from the same process in which you started it.
```

Comment:
i find this sentence hard to read. too long. also i feel like we should have a stronger recommendation for async start. care to take another try at this?

Reply:
Good feedback, rewrote this paragraph and broke it down more with an explicit leaning towards async.